react/features/calendar-sync/middleware.js

```diff
@@ -66,11 +66,9 @@ CALENDAR_ENABLED
                     && store.dispatch(addKnownDomains(knownDomains));
             }
 
-            const result = next(action);
-
             _fetchCalendarEntries(store, false, false);
 
-            return result;
+            return next(action);
```

Comment:
is this change necessary?

Reply:
I want the calendar entries to be available before the APP_WILL_MOUNT action bubbles all the way up so then there is only a slight delay between rendering the welcome screen and getting the entries

Reply:
👍